scripts/surface_template_full_group_pipeline_V1.m

```diff
@@ -2,7 +2,7 @@ function surface_template_full_group_pipeline_V1()
  % Example for the template- and surface-based full pipeline, using the
```

Comment:
This script should not change. Please revert your changes.

Reply:
done in 3bd2d62